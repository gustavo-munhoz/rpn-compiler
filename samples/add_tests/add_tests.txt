(100 200 +)
(123.5 456.25 +)
(-300 150 +)
(10 -9 +)
(-1000 250 +)
(1024 512 +)
(-512.5 1023.5 +)
(327.5 -100.25 +)
(-250.75 -100.125 +)
(999.5 -0.5 +)
(12.5 0 +)
(-100 0 +)
(0 0 +)
(500 -500 +)
(1.25 -1.25 +)
(60000 5000 +)
(65000 1000 +)
(-65504 1 +)
(-60000 -6000 +)
(0.00006103515625 -0.000000059604644775390625 +)
(0.00000011920928955078125 -0.000000059604644775390625 +)