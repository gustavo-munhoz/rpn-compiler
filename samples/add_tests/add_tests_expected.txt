0x5CB0
0x6088
0xD8B0
0x3C00
0xE1DD
0x6600
0x5FFC
0x5B1A
0xDD7B
0x63CE
0x4A40
0xD640
0x0000
0x0000
0x0000
0x7BEE
0x7C00
0xFBFF
0xFC00
0x03FF
0x0000